travis/stashcache-cache-config/Authfile

```diff
@@ -1,3 +1,3 @@
 u * /user/ligo -rl \
-    /stashcache-travis-ci-test rl \
+    / rl \
```

Comment:
I feel a little bit uncomfortable with this. We never tell any of the origins to authorize `\` they normally just authorize the part of the namespace they serve. I guess the idea here is whoever uses this will never have to do anything with Authfile and I guess that is fine.

Reply:
Yeah, I don't think we ever want this for an origin/cache that's ever meant to serve actual data but think this is fine for testing purposes.